control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1888,6 +1888,15 @@ func (r *HostedControlPlaneReconciler) reconcileOAuthServer(ctx context.Context,
 		return fmt.Errorf("failed to reconcile oauth server config: %w", err)
 	}
 
+	pdb := manifests.OAuthServerPodDisruptionBudget(hcp.Namespace)
+	if result, err := r.CreateOrUpdate(ctx, r, pdb, func() error {
+		return oauth.ReconcilePodDisruptionBudget(pdb, p)
+	}); err != nil {
+		return fmt.Errorf("failed to reconcile oauth pdb: %w", err)
+	} else {
+		r.Log.Info("Reconciled oauth pdb", "result", result)
```

Comment:
Nit: Do we in general want to log this? It seems like it is going to be very spammy

Reply:
Now logging at V=2